dnsmasq/DOCS.md

```diff
@@ -32,6 +32,15 @@ forwards:
 hosts:
   - host: home.mydomain.io
     ip: 192.168.1.10
+srv-hosts:
+  - srv: _ldap._tcp.pdc._msdcs.mydomain.io
+    host: dc.mydomain.io
+    port: 389
+    priority: 0
+    weight: 100
+extra-options:
+  - option: "domain=mydomain.io"
+  - option: "mx-host=mx.mydomain.io"
```

Comment:
That is a bit dump. Please make a better logic and expose not raw config options out

Reply:
Why we don't want expose all raw options for dnsmasq?